node/src/components/rest_server.rs

```diff
@@ -85,13 +85,16 @@ pub(crate) struct RestServer {
     /// The task handle which will only join once the server loop has exited.
     #[data_size(skip)]
     server_join_handle: Option<JoinHandle<()>>,
+    /// The instant at which the node has started.
+    node_startup_time: Instant,
```

Comment:
Can we rename this variable `node_startup_instant` throughout please?

Reply:
Done.